FirstHomework/Task1.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func main() {
+	array := []int{9, 7, 2, 5, 3, 6}
+	fmt.Println(median(array))
+}
+
+func median(i []int) float64{
+	sort.Ints(i)
+	var med float64
+
+	if len(i) % 2 == 1{
+		med = float64(i[len(i) / 2])
+		return med
+
+	} else {
```

Comment:
Do we need `else` here? If the condition above isn't met, the flow will pass by the section.
I would prefer:
```
if len(i) % 2 == 1 {
        return float64(i[len(i) / 2])
}

return ...
```